packages/frontend/src/assets/logos/dexes/thirdtrade.tsx

```diff
@@ -0,0 +1,31 @@
+export function ThirdTradeLogo(props: any) {
+    return (
+        <svg
+            width="36.285999"
+            height="36.285999"
+            viewBox="0 0 193.52532 45.861471"
+            version="1.1"
+            {...props}
+        >
+            <g
+                id="logo"
+                stroke="none"
+                stroke-width="1"
+                fill="none"
+                fill-rule="evenodd"
```

Comment:
We should use `camelCase` to avoid console errors.

Reply:
Addressed in 107f7c977945af6605a7e2c90dbbd86e8e793ba6